README.md

```diff
@@ -12,12 +12,12 @@
 
 Powered by [Starlette](https://www.starlette.io/). That `async` declaration is optional. [View documentation](https://responder.readthedocs.io).
 
-This gets you a ASGI app, with a production static files server pre-installed, jinja2 templating (without additional imports), and a production webserver based on uvloop, serving up requests with gzip compression automatically.
+This gets you an ASGI app, with a production static files server pre-installed, jinja2 templating (without additional imports), and a production web server based on up loop, serving up requests with gzip compression automatically.
 
 
 ## Testimonials
 
-> "Pleasantly very taken with python-responder. [@kennethreitz](https://twitter.com/kennethreitz) at his absolute best." —Rudraksh M.K.
+> "Pleasantly very taken with python-responder. [@kennethreitz](https://twitter.com/kennethreitz) at his absolute best." —Rudraksha M.K.
```

Comment:
I think it the name is right https://www.google.com/search?client=ubuntu&channel=fs&q=Rudraksh+M.K&ie=utf-8&oe=utf-8